001-core.php

```diff
@@ -9,6 +9,16 @@
 
 require_once( __DIR__ . '/001-core/privacy.php' );
 
+/**
+ * Disable current theme validation
+ *
+ * By default, WordPress falls back to a default theme if it can't find
+ * the active theme. This is undesirable because it requires manually
+ * re-activating the correct theme and can lead to data loss in the form
+ * of missing sidebar widgets.
```

Comment:
"...can lead to data loss in the form of deactivated widgets and menu location assignments."

- Not all widgets are in a _sidebar_ (front end appearance), but they are in widget areas. 
- They don't get lost completely; they get moved to the Inactive section of the widgets page.
- Menu locations assignments are another practical thing they would otherwise need to fix.